### DISCUSSION TITLE
Start-up question

### DISCUSSION THREAD

**Discussion by educhaj** [mattdodge/yahoofantasy#31] [Q&A]
First off, this is great! I have been looking for a way to do this. However, when I try to run the match-ups I get "'League' object has no attribute 'start_week'." Any idea where I am going wrong? Thanks again!

**Comment by mattdodge** [comment]
Hey @educhaj I'd be happy to help - do you have a code snippet or example I could take a look at?

**Reply by educhaj** [reply]
Thanks! I've been writing it in Atom and running it in Terminal. I could get the teams and players to work, but I keep getting that error on the matchups. I copied this code from my Atom file: 

from yahoofantasy import Context

ctx = Context()
leagues = ctx.get_leagues('mlb', 2022)
for league in leagues:
    print(league.name + " -- " + league.league_type)

from yahoofantasy import League

league = League(ctx, '412.l.91360')  # Use a manual league ID or get it from league.id above
for team in league.teams():
    print(f"Team Name: {team.name}\tManager: {team.manager.nickname}")

week_3 = league.weeks()[2]
for matchup in week_3.matchups:
    print(f"{matchup.team1.name} vs {matchup.team2.name}")

matchup = week_3.matchups[0]
print(f"{matchup.team1.name}\tvs\t{matchup.team2.name}")
for team1_stat, team2_stat in zip(matchup.team1_stats, matchup.team2_stats):
    print(f"{team1_stat.value}\t{team1_stat.display}\t{team2_stat.value}")

**Comment by mattdodge** [comment]
Aha, this appears to be because you are using the manual league ID to create your league, rather than grabbing one of the results from the `ctx.get_leagues` call. This is admittedly a bit confusing, and I should probably fix that.

But, to get you going faster, instead of doing `league = League(ctx, '412.l.91360')` try doing this:
```python
league = next(l for l in leagues if l.id == "412.l.91360")
```

That will basically grab the league you are looking for out of the list of fetched leagues, which will have much more info already fetched.

**Reply by educhaj** [reply]
That did it! Thanks!
